parser/sql/dialect/oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/OracleStatementVisitor.java

```diff
@@ -850,4 +943,775 @@ public final ASTNode visitDataTypeLength(final DataTypeLengthContext ctx) {
     protected String getOriginalText(final ParserRuleContext ctx) {
         return ctx.start.getInputStream().getText(new Interval(ctx.start.getStartIndex(), ctx.stop.getStopIndex()));
     }
+    
+    @Override
+    public ASTNode visitSelect(final SelectContext ctx) {
```

Comment:
Can you use `new OracleDMLStatementVisitor().visit()`?

Reply:
Besides, we should merge `private final Collection<ParameterMarkerSegment> parameterMarkerSegments = new LinkedList<>();` if we use `new OracleDMLStatementVisitor().visit()`.

Reply:
OK

Reply:
The codes have been refactored.